openfl/federated/plan/plan.py

```diff
@@ -566,7 +605,7 @@ def get_serializer_plugin(self, **kwargs):
             self.serializer_ = Plan.build(serializer_plugin, kwargs)
         return self.serializer_
 
-    def restore_object(self, filename):
+    def restore_object(self, filename: str) -> Any:
```

Comment:
We can also accept `os.PathLike[str]` for `filename`.

Reply:
I guess we can, but I can't find where do we use it?

Reply:
It will cause error: 'ABCMeta' object is not subscriptable